.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  test:
```

Comment:
Better 👍  When this is merged please update the branch protection rules to require the "test" action to succeed instead of "build".